<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenericValueForm</class>
 <widget class="QWidget" name="GenericValueForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="mold::StatusBarForm" name="statusBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>215</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="generic_unit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>115</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Unit</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="mold::RoomHeaderForm" name="header" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="mold::InvisibleClickableButton" name="myLovelyButton">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>175</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="generic_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>53</y>
     <width>320</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="generic_value">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>106</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QWidget" name="motion" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/settings/icons/motion.svg);</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mold::InvisibleClickableButton</class>
   <extends>QPushButton</extends>
   <header>gui/InvisibleClickableButton.hpp</header>
  </customwidget>
  <customwidget>
   <class>mold::StatusBarForm</class>
   <extends>QWidget</extends>
   <header>gui/StatusBarForm.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mold::RoomHeaderForm</class>
   <extends>QWidget</extends>
   <header>gui/RoomHeaderForm.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
